### ISSUE TITLE
Investigate BBOX for when input is Adm1 or Adm2

### ISSUE THREAD

**Issue by ClaraIV** [kartoza/GEEST2#374]
If users are using ADM1 or ADM2 as input files in the analysis, the below happens -> could this be avoided/restricted to only outline or could a message be triggered to inform of the consequences?

![Image](https://github.com/user-attachments/assets/12af1e63-0dff-4886-9327-00450464aa13)

**Comment by timlinux** [comment]
Hi @ClaraIV could you clarify the expected behaviour you would like to see here?  Should the union of polygons be used, dissolving interior boundaries and only using the outer boundary?

**Comment by ClaraIV** [comment]
yes. I believe this is a consequence of the innovation with the different bbox. But it doesnt make sense to have different bbox for sub-admins, right? As such, only the union of polygons should be used, or perhaps we can restrict the usage of adm1/2/3 somehow? happy to discuss on Wed and hear your thoughts. Ideally people would input only ADM0, if this can be made clear somewhere or enforced that would be great

**Comment by timlinux** [comment]
@ClaraIV I tried to make this requirement clear in the the project creation panel here:

![Image](https://github.com/user-attachments/assets/c410bfa8-333a-49de-9324-72a48c41e0b4)


I'm going to mark this as taken care of for now, or let me know if you think there should be further action.

Thanks
